tldw_Server_API/app/api/v1/endpoints/admin/admin_tools.py

```
@@ -0,0 +1,185 @@
+    "/mcp/tool_catalogs/{catalog_id}",
+    summary="Delete MCP tool catalog (admin)",
+    description=(
+        "Delete a catalog by id. Entries are removed via ON DELETE CASCADE.\n\n"
+        "RBAC: Admin-only.\n\n"
+        "Scope: Works for any catalog (global/org/team)."
+    ),
+    dependencies=[Depends(require_roles("admin"))],
+)
+async def delete_tool_catalog(
+    catalog_id: int,
+    db=Depends(get_db_transaction),
```

Ruff (0.14.14)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)